go.mod

```diff
@@ -37,12 +37,15 @@ require (
 	k8s.io/cluster-bootstrap v0.22.3
 	k8s.io/component-base v0.22.3
 	k8s.io/controller-manager v0.22.3
+	k8s.io/klog v1.0.0
```

Comment:
please do not import `k8s.io/klog` package, use `k8s.io/klog/v2` instead.

Reply:
done